cli/flags.rs

```diff
@@ -1731,7 +1750,8 @@ mod tests {
         subcommand: DenoSubcommand::Fmt {
           ignore: vec![],
           check: false,
-          files: vec!["script_1.ts".to_string(), "script_2.ts".to_string()]
+          files: vec!["script_1.ts".to_string(), "script_2.ts".to_string()],
+          verbose: false,
```

Comment:
Please test `--verbose` flag in one test for fmt and one for lint

Reply:
Done :)